src/controllers/user.ts

```diff
@@ -34,7 +35,7 @@ class UsersController {
     }
   }
 
-  async getUsers(req: Request, res: Response) {
+  async getUsers(req: Request<null, null, null>, res: Response) {
```

Comment:
There is no sense to provide null just 
```suggestion
  async getUsers(req: Request, res: Response) {
```